### ISSUE TITLE
Deno completion stopped work

### ISSUE THREAD

**Issue by igorbrasileiro** [nvim-lua/kickstart.nvim#350]
> I'm not a nvim expert, help me to help you 😄 

## Describe the bug
I'm using deno with the latest version of kickstart, with changes removing typescript/tsx at treesitter and added deno to mason.
After this upgrade, my completion stopped work.

## To Reproduce

1. open the nvim
2. open a typescript file
3. define a variable
4. type prefix of variable of 3

code example:
```tsx
import { useSignal } from "@preact/signals"
function MyPreactComponent() {
  const mySignal = useSignal(0);
  const isHigherThanTen = mySign // expected behavior is to show mySignal
}
```

## Desktop

- OS: macos
- Terminal: iterm2

## Neovim Version

NVIM v0.9.1
Build type: Release
LuaJIT 2.1.0-beta3
system vimrc file: "$VIM/sysinit.vim"
fall-back for $VIM: "/opt/homebrew/Cellar/neovim/0.9.1/share/nvim"

deno 1.34.3

**Comment by pudgypaw** [comment]
Just saw this, I forked this from the main project.
Please consider opening a bug over on [their repo](https://github.com/nvim-lua/kickstart.nvim)

Thanks for the report!

**Comment by feoh** [comment]
I don't know what a deno is so I have no idea as to how to help you.

Please let us know if you fix the problem.

**Comment by igorbrasileiro** [comment]
TLDR; I fixed with completions from buffer

Sorry, I was studying about LSP, completions and deno, and I realized that this feature of completions text from buffer doesn't belongs to deno, but the editor.

I achieved this feature with a custom plugin for buffer completions https://github.com/hrsh7th/cmp-buffer

**Comment by feoh** [comment]
Glad you fixed it and are all set!
